lib/components/artist/related_artists/index.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react-native';
+const { ScrollView, StyleSheet, View } = React;
+
+import colors from '../../../../data/colors';
+import SerifText from '../../text/serif';
+import ImageView from '../../opaque_image_view';
+import RelatedArtist from './related_artist'
+
+class RelatedArtists extends React.Component {
+  render() {
+    const artists = this.props.artists
+    return (
+      <View style={styles.container}>
+        <SerifText style={styles.heading}>Related Artists</SerifText>
+        <View style={styles.artistContainer}>
+          { artists.map(artist => <RelatedArtist key={artist.id} artist={artist} /> )}
+        </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    borderTopWidth: 1,
+    borderTopColor: colors['gray-regular'],
+    flexDirection: 'column',
+  },
+  artistContainer: {
+    flexWrap: 'wrap',
+    flexDirection: 'row',
+    justifyContent: 'space-around',
+    alignItems: 'stretch',
+    marginTop: 20,
+    marginLeft: -10,
```

Comment:
I'll find a better way to do this here and elsewhere because I'm still trying to figure out the default side margins.